{
    "config": {
        "error": {
            "cannot_connect": "Fall\u00f3 la conexi\u00f3n",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Host",
                    "password": "Contrase\u00f1a",
                    "port": "Puerto",
                    "username": "Usuario",
                    "web_port": "Puerto web (para el servicio de visita)"
                }
            }
        }
    }
}